EAD/EADSession1/Contact.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+
```

Comment:
remove extra lines

Reply:
For building the UI we have to use HTML only.